package.json

```diff
@@ -34,9 +34,9 @@
     "ocis-kill": "killall ocis",
     "docker-kill": "docker kill web-tests-selenium web-tests-redis",
     "testing-app": "if [ -d tests/testing-app ]; then (cd tests/testing-app && git pull && pwd); else git clone --depth 1 https://github.com/owncloud/testing.git tests/testing-app; fi;",
-    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis/ocis; make clean; make",
+    "build-ocis": "if [ -d tests/ocis ]; then (cd tests/ocis && git reset --hard master && git pull); else git clone --depth 1 https://github.com/owncloud/ocis.git tests/ocis; fi; cd tests/ocis/ocis; make clean; make generate build",
```

Comment:
Adding `make generate build` is probably "a good thing". It seems like it is not related to this PR, but might be good to do anyway.

Reply:
make alone does nothing. Problem was that the package.json items were no working at all.